lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb

```diff
@@ -122,26 +122,26 @@ def self.prompt_for_version
       end
 
       # 
-      def self.create_fastlane_session()
+      def self.create_fastlane_session
         require 'fastlane/plugin/cryptex'
 
-        UI.message "Generating a new FASTLANE_SESSION."
+        UI.message "Generating a new fastlane session."
 
         fastlane_session_git_url =  ENV["FASTLANE_ENV_GIT_URL"]
         fastlane_session_username = ENV["FASTLANE_ENV_USERNAME"]
-
+        fastlane_session_password = ENV["CRYPTEX_PASSWORD"]
         spaceship_cookie_path = "#{File.expand_path('~')}/.fastlane/spaceship/#{fastlane_session_username}/cookie"
-        
-        if File.exists?(spaceship_cookie_path)
-          system `rm #{spaceship_cookie_path}`
-        end
 
         if !fastlane_session_username;
           UI.user_error!("No FASTLANE_ENV_USERNAME var at <root>/fastlane/.env\nFASTLANE_ENV_USERNAME is used to authenticate with the App Store for iOS releases.")
         elsif !fastlane_session_git_url;
           UI.user_error!("No FASTLANE_ENV_GIT_URL var at <root>/fastlane/.env\nFASTLANE_ENV_GIT_URL is used to store the App Store Connect session to upload releases on CI.")
+        elsif !fastlane_session_password;
```

Comment:
side-note (just FYI, non-blocking) we shouldn't need semicolons here

Reply:
HAHAHAHAHAHA

Reply:
I can't help it

Reply:
Future me will remove.